File: src/main/java/com/sotasan/decompiler/menus/MenuBar.java
package com.sotasan.decompiler.menus;

import com.formdev.flatlaf.extras.components.FlatMenuBar;
import com.sotasan.decompiler.menus.help.Help;
import com.sotasan.decompiler.menus.file.File;

public class MenuBar extends FlatMenuBar {

    public MenuBar() {
        add(new File());
        add(new Help());
    }

}

File: jvm/src/main/kotlin/moe/sota/decompiler/jvm/menus/MenuBar.kt
package moe.sota.decompiler.jvm.menus

import com.formdev.flatlaf.extras.components.FlatMenuBar
import moe.sota.decompiler.jvm.menus.file.File
import moe.sota.decompiler.jvm.menus.help.Help

class MenuBar(file: File, help: Help) : FlatMenuBar() {
    init {
        add(file)
        add(help)
    }
}
